app/test_main.py

```diff
@@ -1,3 +1,21 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "value1,value2,result",
+    [
+        (-3, -3, [0, 0]),
+        (10, 10, [0, 0]),
+        (16, 16, [1, 1]),
+        (24, 24, [2, 2]),
+        (100, 100, [21, 17]),
+        (32, 94, [4, 16])
+    ])
+def test_gethuman_age(value1: int, value2: int, result: list) -> None:
+    assert get_human_age(value1, value2) == result
+
+
+def test_not_supported_type_of_data() -> None:
+    with pytest.raises(TypeError):
+        get_human_age("a", "b")
```

Comment:
Use `@pytest.mark.parametrize` here as well and test against not only strings, but lists, dicts, tuples, sets, e.g.